content/en/docs/concepts/security/index.md

```diff
@@ -545,11 +548,12 @@ an authorization policy to the workloads to enforce access control.
 For workloads without authorization policies applied, Istio doesn't enforce
 access control allowing all requests.
 
-Authorization policies support both `ALLOW` and `DENY` actions. The deny
-policies take precedence over allow policies. If any allow policies are applied
-to a workload, access to that workload is denied by default, unless explicitly
-allowed by the rule in the policy. When you apply multiple authorization
-policies to the same workload, Istio applies them additively.
+Authorization policies support `ALLOW`, `DENY` and `CUSTOM` actions and the evaluation order is `CUSTOM`, `DENY` and
```

Comment:
AUDIT is also supported. I guess you might be intentionally not mentioning audit here. but as an user/developer, I think it's interesting and good to know the enforcement order of the audit, relative to other actions.

Reply:
yes, I left it intentionally, I think it should be added as a separate section in the end of the concept page, in parallel with the `AuthN` and `AuthZ`. cc @xulingqing